runtime/src/lib.rs

```diff
@@ -112,7 +112,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave `spec_version` as
 	// is and increment `impl_version`.
 	spec_version: 37,
-	impl_version: 37,
+	impl_version: 38,
```

Comment:
this will be release 38? runtime version 38

Reply:
it's only `imp_version` change but yeah might be best to leave it at `37`